docs/tutorials/embedding/MASE.ipynb

```diff
@@ -0,0 +1,374 @@
+    "\n",
+    "There is clear block structure to the graphs, and we see that the second, lower right, block of $G_1$ has more edges than that of $G_2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspy.plot import heatmap\n",
+    "\n",
+    "heatmap(G1, figsize=(7,7), title=\"Visualization of Graph 1\")\n",
+    "heatmap(G2, figsize=(7,7), title=\"Visualization of Graph 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed the two graphs using MASE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MASE fits a sample of graphs to the common-subspace independent edge (COSIE) model for multiple graphs. The COSIE model assumes that each graph is generated from a common invariant subspace, $V$, but with potentially different connection probabilities between each vertex, given by score matrices $R^{(i)}$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When we use all of the default parameters, ``MultipleASE()`` will automatically determine the optimal embedding dimension.  The ``fit_transform()`` method returns the estimated invariant subspace $\\hat{V}$, which we can see has 50 entries corresponding to the 50 vertices embedded in 2 dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspy.embed import MultipleASE as MASE\n",
+    "\n",
+    "embedder = MASE()\n",
+    "V_hat = embedder.fit_transform([G1, G2])\n",
+    "print(V_hat.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize Common Latent Subspace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the two graphs have clear block structures, with higher within-block probabilities than between-block, we see two distinct \"clusters\" when we visualize the latent positions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(10,10))\n",
+    "ax.scatter(V_hat[:25, 0], V_hat[:25, 1], c=\"blue\", label=\"Block 1\")\n",
+    "ax.scatter(V_hat[25:, 0], V_hat[25:, 1], c=\"red\", label=\"Block 2\")\n",
+    "ax.legend()\n",
+    "ax.set_title(\"Common Latent Subspace from MASE Embedding\", fontsize=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Recover Probability Matrices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since ``MultipleASE()`` stores estimates of the individual score matrices $\\hat{R}^{(i)}$ as well as the common invariant subspace $\\hat{V}$, we can recover an estimate of the probability matrix for each graph \n",
```

Comment:
what these things mean should be explained above when you talk about what MASE does.